### PR TITLE
Anu dev

### PR DESCRIPTION
**Pull request by anuragjha**
stock extractor service

### CODE REVIEW

File: sam-local.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ Resources:
       CodeUri: build/exploded
       MemorySize: 320
       Policies: AWSLambdaBasicExecutionRole
-      Timeout: 15
+      Timeout: 29
```
Comment (hiepcs) [inline sam-local.yaml]: Why you use 29 instead of 30? Any reason for that?

File: src/main/java/usf/sdlc/form/StockHistoryForm.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package usf.sdlc.form;
+
+public class StockHistoryForm {
```
Comment (hiepcs) [inline src/main/java/usf/sdlc/form/StockHistoryForm.java]: Will you use this class or not, because i saw you comment out all

**Comment by anuragjha** [comment]
I saw in AWS that it is 29 by default, that’s why I changed it to 29

Thanks,
Anurag Jha

File: sam-local.yaml (hunk 1 above)
Comment (anuragjha) [reply sam-local.yaml]: No specific reason

File: src/main/java/usf/sdlc/form/StockHistoryForm.java (hunk 2 above)
Comment (anuragjha) [reply src/main/java/usf/sdlc/form/StockHistoryForm.java]: No, I will not be using this class

**Pull request merged**